_includes/presentations.html

```diff
@@ -6,6 +6,7 @@ <h2>Presentations</h2>
                 <hr class="title-line check-calendar">
             </div>
             <ul class="presentations">
+                <li><a href="https://www.twitch.tv/videos/2071508641">TinkerPop Wide: Q&A with JanusGraph</a>, Oleksandr Porunov, Boxuan Li, & Stephen Mallette - 2024.02.23</li>
```

Comment:
I think the recordings are only available on Twitch until they are uploaded to [the YouTube channel](https://www.youtube.com/@apachetinkerpop). I at least can't see any older recordings on that channel.

Reply:
Hmm, interesting.
 @spmallette do you know if the recording will be removed from Twitch after being uploaded to YouTube? If yes, then we should simply update the link when it’s on YouTube.

Reply:
yes, they leave twitch. i will get it uploaded to youtube soon. just haven't had a chance to focus on that yet.

Reply:
Thank you! Please, ping us when it’s on YouTube and we will update the link to the recording.